content/en/continuous_integration/guides/code_coverage.md

```diff
@@ -77,6 +77,103 @@ When code coverage is available, the Datadog Tracer (v2.31.0 or later) reports i
 
 If you are using [Coverlet][101] to compute your code coverage, indicate the path to the report file in the `DD_CIVISIBILITY_EXTERNAL_CODE_COVERAGE_PATH` environment variable when running `dd-trace`. The report file must be in the OpenCover or Cobertura formats. Alternatively, you can enable the Datadog Tracer’s built-in code coverage calculation with the `DD_CIVISIBILITY_CODE_COVERAGE_ENABLED=true` env variable.
 
+### Advanced options (Supported through .runsettings)
```

Comment:
nit: "supported", instead of "Supported"

Reply:
changed in [d49558d](https://github.com/DataDog/documentation/pull/19630/commits/d49558dc228b44592b3d63cc15163d9b322ecc45)